plugins/mcreator-core/procedures/entity_execute_command.json

```diff
@@ -2,7 +2,7 @@
   "args0": [
     {
       "type": "input_value",
-      "name": "command",
```

Comment:
Here you renamed existing parameters, which is not backwards compatible. Check all cases of this and revert them